recipes-bsp/u-boot/u-boot-imx_2021.04.bb

```diff
@@ -38,4 +38,4 @@ do_deploy:append:mx8m() {
 }
 
 PACKAGE_ARCH = "${MACHINE_ARCH}"
-COMPATIBLE_MACHINE = "(mx6|mx7|mx8|use-mainline-bsp)"
+COMPATIBLE_MACHINE = "(mx6-generic-bsp|mx7-generic-bsp|mx8-generic-bsp)"
```

Comment:
Not sure if the `*-generic-bsp` is required here, perhaps `*-nxp-bsp` would be sufficient - _Mainline BSP_ does use `u-boot-fslc` recipe instead.

Reply:
Generally yes, but I used `generic` here because we eventually use `imx` one while converting a machine mainline. Also, other BSP might be using this as the `mainline` was accepted.